packages/xrpl/test/integration/transactions/nftokenMint.test.ts

```diff
@@ -91,4 +96,63 @@ describe('NFTokenMint', function () {
     },
     TIMEOUT,
   )
+
+  it(
+    'test with Amount',
+    async function () {
+      const tx: NFTokenMint = {
+        TransactionType: 'NFTokenMint',
+        Account: testContext.wallet.address,
+        URI: convertStringToHex('https://www.google.com'),
+        NFTokenTaxon: 0,
+        Amount: xrpToDrops(1),
+        Expiration: unixTimeToRippleTime(Date.now() + 1000 * 60 * 60 * 24),
+        Destination: destinationWallet.address,
+      }
+      const response = await testTransaction(
+        testContext.client,
+        tx,
+        testContext.wallet,
+      )
+      assert.equal(response.type, 'response')
+
+      const txRequest: TxRequest = {
+        command: 'tx',
+        transaction: hashSignedTx(response.result.tx_blob),
+      }
+      const txResponse = await testContext.client.request(txRequest)
+
+      assert.equal(
+        (txResponse.result.meta as TransactionMetadata).TransactionResult,
```

Comment:
This change to the `tx` model ought to fix your typing issues here:
```
  meta_blob?: Version extends typeof RIPPLED_API_V2 ? string : never
  meta?: Version extends typeof RIPPLED_API_V2
    ? TransactionMetadata<T>
    : TransactionMetadata<T> | string
```

Reply:
@mvadari  This tests uses rippled APIv2 (default version, if `api_version` field is not specified). Is there a need to multiplex between the APIv1 and APIv2 types? How is this useful?

Reply:
The `tx` model is slightly incorrect - the casting shouldn't be necessary, and this change should fix that.

Reply:
@tequdev if you'd like to address this comment, we can wait.